src/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -447,13 +455,13 @@ public abstract partial class JsonAttribute : System.Attribute
     public abstract partial class JsonConverter
     {
         internal JsonConverter() { }
-        public abstract bool CanConvert(System.Type type);
+        public abstract bool CanConvert(System.Type typeToConvert);
```

Comment:
Same here, @steveharter - the source and the ref didn't agree. I have updated the parameter name to match what was API approved.